src/frontend/lib/main.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:apa_app/views/home.dart';
+import 'package:apa_app/views/addfield.dart';
+import 'package:apa_app/views/loading.dart';
+import 'package:apa_app/views/myfields.dart';
+import 'package:apa_app/views/flydrone.dart';
+import 'package:apa_app/views/settings.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+Future main() async {
+  // Allows env vars to be used in source code
+  await dotenv.load(fileName: "lib/.env");
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
+
+  const MyApp({super.key});
+
+  // This widget is the root of your application.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Autonomous Precision Agriculture using UAVs',
+      theme: ThemeData(
+        primarySwatch: Colors.lightGreen,
+      ),
+      initialRoute: '/',
+      routes: {
+        '/': (context) => const MainPage(),
```

Comment:
We could add a Page for invalid routes. Anything outside of the valid routes will go to this "Error" page.

Reply:
Sounds good, however shall we maybe add that later? (as it is not necessarily related to this issue) ;)